dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/SdkFeatureTest.kt

```diff
@@ -130,6 +149,61 @@ internal class SdkFeatureTest {
         assertThat(testedFeature.uploader).isInstanceOf(DataOkHttpUploader::class.java)
     }
 
+    @Test
+    fun `𝕄 use the storage frequency if set 𝕎 initialize()`(forge: Forge) {
+        // Given
+        val fakeUploadFrequency = forge.aValueFrom(UploadFrequency::class.java)
+        val fakeStorageConfig = mockWrappedFeature.storageConfiguration
+            .copy(uploadFrequency = fakeUploadFrequency)
+        whenever(mockWrappedFeature.storageConfiguration)
+            .thenReturn(fakeStorageConfig)
+        // When
+
+        testedFeature.initialize(appContext.mockInstance)
+
+        // Then
+        assertThat(testedFeature.uploadScheduler)
+            .isInstanceOf(DataUploadScheduler::class.java)
+        val dataUploadRunnable = (testedFeature.uploadScheduler as DataUploadScheduler).runnable
+        assertThat(dataUploadRunnable.uploadFrequency).isEqualTo(fakeUploadFrequency)
+    }
+
+    @Test
+    fun `𝕄 initialize the storage 𝕎 initialize()`() {
```

Comment:
with core batch size probably is missing in the name?

Reply:
I am considering this as default initialization so wanted to keep it as that in the test description